fix(ui): remove glossy <select> effect in Safari

SecondThundeR opened the pull request:
This small CSS fix removes the "glossy" effect of <select> tags in Safari

SecondThundeR commented:
Before:
<img width="1212" alt="image" src="https://user-images.githubusercontent.com/36604233/218043821-acd3734f-f7b7-49c5-8b2e-dc151929d334.png">

After:
<img width="1041" alt="image" src="https://user-images.githubusercontent.com/36604233/218043848-7f01cb50-a947-4a5e-9f5d-db541e009afd.png">

aarthificial commented on packages/ui/src/index.scss:
```diff
@@ -25,6 +25,10 @@
   font-size: 14px;
   accent-color: var(--theme);
 
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  appearance: none;
```
Can you check if just setting:
```suggestion
  -webkit-appearance: caret;
```
wouldn't work?
Setting `appearance` to `none` removes the dropdown arrow from all browsers 😞

SecondThundeR commented:
Unfortunately, option with `caret` returns that style, I'll try to see what I can do to keep dropdown arrow as well

SecondThundeR commented:
I have currently found some solution that looks good. By using `appearance: none` only on the select tag and adding a custom dropdown icon via the `background` property (I took ChevronRight as a base and rotated it 90 degrees) it solves the original problem and makes the icon itself prettier than the standard browser icon.

The only thing is that there are still cross-browser problems (Now I believe why Safari is called the new IE). Since after `appearance: none`, the width of select becomes different (on Chrome and Safari the width isn't the same), the width of input in playback controls also breaks. As I understood from some "experiments", because of the grid layout, the min-content parameter in grid-template-columns takes the width of select (as I guess), which is smaller than input and because of this the text in input is cut off.

Of course, in theory, we could add a fixed width select in playback controls, which would eventually make it look almost the same in both browsers.

So I'd like to figure out if I should do such a "hack" for the sake of the great UI, or scrape this idea for now and leave the controls in Safari the way they are (Which of course I wouldn't want to, because the new dropdown icon looks more interesting after all)

Examples of change:
<img width="1123" alt="image" src="https://user-images.githubusercontent.com/36604233/218090163-e420141e-c477-4c9a-813d-e6bdeaa633ee.png">

<img width="1318" alt="image" src="https://user-images.githubusercontent.com/36604233/218090127-e7b0edee-cc3b-4f48-abd8-306ec621b104.png">

_different sizes of select in both browsers_
<img width="363" alt="image" src="https://user-images.githubusercontent.com/36604233/218090568-b772405f-2263-4c78-b071-38e9f179ceaf.png">

<img width="298" alt="image" src="https://user-images.githubusercontent.com/36604233/218090600-81d0b84a-c4e4-4705-af65-4b438e61655d.png">

aarthificial commented:
I think adding a fixed width, or at least a `min-width` on the selects in the playback controls is a good idea. Right now their size is not really well defined so it's not surprising that it varies between browsers.

SecondThundeR commented:
After a little poking around, I was able to figure out that the cause of the weird width is also the size property of input, which (again) works [differently in both browsers](https://stackoverflow.com/a/1481084). So I removed it in the `PlaybackControls` component and added a fixed width for `.controls` select and input. Now the width looks great in both browsers

<img width="1385" alt="image" src="https://user-images.githubusercontent.com/36604233/218256098-a2353918-2def-4ac0-9d3a-198df512a2af.png">
<img width="1326" alt="image" src="https://user-images.githubusercontent.com/36604233/218256103-db551c9c-90f6-4bcc-b53f-2ab3cea71378.png">

aarthificial commented:
Awesome, thanks for the PR!
